.github/workflows/deployment-images.yaml

```diff
@@ -38,15 +44,20 @@ on:
         required: true
 
 jobs:
-  set-deployment-image-name:
+  set-image-names:
     runs-on: ${{ inputs.runs-on }}
     outputs:
       name: ${{ steps.set-deployment-image-name.outputs.name }}
+      test-name: ${{ steps.set-deployment-test-image-name.outputs.test_name }}
     steps:
       - id: set-deployment-image-name
         run: |
           name=$([[ "${{ inputs.deployment-image-name }}" != "" ]] && echo ${{ inputs.deployment-image-name }} || echo ${{ github.repository }} | cut -f2 -d/)
           echo "name=$name" >> $GITHUB_OUTPUT
+      - id: set-deployment-test-image-name
+        run: |
+          test_name=$([[ "${{ inputs.deployment-test-image-name }}" != "" ]] && echo ${{ inputs.deployment-test-image-name }} || echo ${{ github.repository }}_tests | cut -f2 -d/)
+          echo "test_name=$test_name" >> $GITHUB_OUTPUT
```

Comment:
we must use underscores here but the output variables names use hyphens.

Reply:
Is this a shell restriction?

Reply:
In other words, `echo "test-name=$test_name" >> $GITHUB_OUTPUT` does not work?

Reply:
correct. I tried it here - https://github.com/rentpath/lead-service/actions/runs/4306337859/jobs/7510130558#step:3:6

Reply:
updated the sh var to be underscores while the output name uses hyphens. tested here - https://github.com/rentpath/lead-service/actions/runs/4306499874/jobs/7511149872#step:4:197